Source/WebCore/rendering/style/StyleFilterImage.cpp

```diff
@@ -165,4 +165,18 @@ void StyleFilterImage::imageChanged(CachedImage*, const IntRect*)
         client->imageChanged(static_cast<WrappedImagePtr>(this));
 }
 
+void StyleFilterImage::didAddClient(RenderElement& renderer)
+{
+    if (!m_image)
+        return;
+    m_image->addClient(renderer);
```

Comment:
Maybe the coding guideline says to use an early return instead, but given the size of the method it seems doing a non-null check instead would be preferable.